lib/grape/validations/attributes_doc.rb

```diff
@@ -28,6 +28,12 @@ def extract_details(validations)
         details[:documentation] = documentation if documentation
 
         details[:default] = validations[:default] if validations.key?(:default)
+
+        return unless validations.key?(:length)
```

Comment:
we'll be adding other validations, so let's not prematurely return, a double `if` is fine, it will get optimized away

```
details[:min_length] = validations[:length][:min] if validations.key?(:length) && validations[:length].key?(:min)
details[:max_length] = validations[:length][:max] if validations.key?(:length) && validations[:length].key?(:max)

```

Reply:
done 👍